cmd/process-agent/app/status.go

```diff
@@ -0,0 +1,225 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package app
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"text/template"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/DataDog/datadog-agent/cmd/process-agent/api"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	ddconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+var httpClient = util.GetClient(false)
+
+const (
+	statusTemplate = `
+==============================
+Process Agent ({{ .Core.AgentVersion }})
+==============================
+
+ Status date: {{ .Date }}
+ Process Agent Start: {{ .Expvars.Uptime }}
+ Pid: {{ .Expvars.Pid }}
+ Go Version: {{ .Core.GoVersion }}
+ Python Version: {{ .Core.PythonVersion }}
+ Build arch: {{ .Core.Arch }}
+ Log Level: {{ .Core.Config.LogLevel }}
+ Enabled Checks: {{ .Expvars.EnabledChecks }}
+
+=========
+Collector
+=========
+
+  Last collection time: {{.Expvars.LastCollectTime}}
+  Docker socket: {{.Expvars.DockerSocket}}
+  Number of processes: {{.Expvars.ProcessCount}}
+  Number of containers: {{.Expvars.ContainerCount}}
+  Process Queue length: {{.Expvars.ProcessQueueSize}}
+  RTProcess Queue length: {{.Expvars.RTProcessQueueSize}}
+  Pod Queue length: {{.Expvars.PodQueueSize}}
+  Process Bytes enqueued: {{.Expvars.ProcessQueueBytes}}
+  RTProcess Bytes enqueued: {{.Expvars.RTProcessQueueBytes}}
+  Pod Bytes enqueued: {{.Expvars.PodQueueBytes}}
+
+`
```

Comment:
I think that's not a bad idea, however a lot of this data is not currently exposed so it may take me a bit of time to implement it. Let me know what you think

Reply:
I think that we can tackle it in a follow-up PR!